### PR TITLE
Quiz

### PR DESCRIPTION
**Pull request by ashwoodaa**
Booglethorp
just a test

### CODE REVIEW

**Comment by ashwoodaa** [comment]
1. added actual html to html file
2. added width to cards
3. background color to whole page

**Comment by ashwoodaa** [comment]
:bomb:  BOOM DONE :boom:

File: index.html [hunk 1]
```diff
@@ -0,0 +1,201 @@
...
+	<title></title>
+	<link rel="stylesheet" href="quiz.css">
+</head>
+<body>
+  <article class="cards">
+    <header class="header">
+      <h1>InsertProductTitle</h1>
+    </header>
+    <div id="firstSection">
+    	<img src="woodBurningPic4.jpg" alt="Dragon">
```
Comment (gregkorte) [inline index.html]: `` tags take on properties of both inline and block level elements. You can satisfy the requirements by wrapping this tag in a `<div>`.

**Comment by gregkorte** [comment]
Your quiz looks good, Amari. Just make that one change, and you are good to merge. :+1:

**Comment by ashwoodaa** [comment]
Thanks!

**Pull request merged**